server/sheet_test.go

```diff
@@ -0,0 +1,66 @@
+package server
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseSheetInfo(t *testing.T) {
+	tests := []struct {
+		filePath          string
+		sheetPathTemplate string
+		want              SheetInfo
+	}{
+		{
+			filePath:          "sheet/test.sql",
+			sheetPathTemplate: "sheet/{{NAME}}.sql",
+			want: SheetInfo{
+				EnvironmentName: "",
+				DatabaseName:    "",
+				SheetName:       "test",
+			},
+		},
+		{
+			filePath:          "sheet/DEV__TEST__test.sql",
+			sheetPathTemplate: "sheet/{{ENV_NAME}}__{{DB_NAME}}__{{NAME}}.sql",
+			want: SheetInfo{
+				EnvironmentName: "DEV",
+				DatabaseName:    "TEST",
+				SheetName:       "test",
+			},
+		},
+	}
+
+	println(filepath.Dir("sheets/{{asd}}/{{Name}}.sql"))
+
+	for _, test := range tests {
+		result, err := parseSheetInfo(test.filePath, test.sheetPathTemplate)
+		if err != nil {
+			t.Errorf("Parse sheet path template error %v.", err)
+		}
+		require.Equal(t, *result, test.want)
+	}
+}
+
+func TestParseBasePathFromTemplate(t *testing.T) {
+	tests := []struct {
+		sheetPathTemplate string
+		want              string
+	}{
+		{
```

Comment:
let's add

1. Only {{DB_NAME}} is missing
2. {{NAME}} is missing
3. not sub directory like  {{NAME}}.sql